modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -101,6 +98,8 @@ def post_appointment(request_object_body)
           convert_appointment_time(new_appointment)
```

Comment:
I find it strange that POST and PUT calls do not call `prepare_appointment` and instead only perform a subset of the post processing. I have kept these separate for now but I'd like to understand whether there's a reason for this difference or whether this was an oversight. I looked through `prepare_appointment` and didn't see any obvious danger so I'm leaning towards calling `prepare_appointment` here as well. But before I do that I'd like to double check my understanding @simiadebowale @cferris32.

Reply:
FYI I tested out calling `prepare_appointment` here instead of the mish-mash of different function calls and all tests pass so I'm leaning towards just calling `prepare_appointment`. However, this would be a behavior change so I'd still like to double check it before I proceed.

Reply:
We decided that further consolidation here requires more discussion and @cferris32 will file a spike to confirm the viability of the change.